dash/dash-renderer/src/components/error/menu/VersionInfo.react.js

```diff
@@ -75,12 +88,19 @@ async function requestDashVersionInfo(config) {
 }
 
 function shouldRequestDashVersion(config) {
-    const showNotificationsLocalStorage =
-        localStorage.getItem('showNotifications');
-    const showNotifications = config.disable_version_check
-        ? false
-        : showNotificationsLocalStorage !== 'false';
-    const lastFetched = localStorage.getItem('lastFetched');
+    let showNotificationsLocalStorage, showNotifications, lastFetched;
+    try {
+        showNotificationsLocalStorage =
+            localStorage.getItem('showNotifications');
+
+        showNotifications = config.disable_version_check
```

Comment:
Can we move that check to the DebugMenu render like: `{!config.disable_version_check && <VersionInfo />}` ?

Reply:
The `VersionInfo` component also shows the current version of dash, which we want to be shown either way